less/about/preferences.less

```diff
@@ -455,6 +455,61 @@ table.sortableTable {
   overflow-y: scroll;
 }
 
+.modal .dialog.paymentHistory .sectionTitle {
+  text-align: left;
+}
+
+span.paymentHistoryButton.browserButton {
+  display: block;
+  color: @braveOrange;
+  font-size: 14px;
+  white-space: nowrap;
+}
+
+.paymentHistoryFooter span.okButton.primaryButton {
+  background-color: gray;
+  float: right;
+  margin-top: -10px;
+  display: inline;
+  padding-top: 5px;
+  padding-left: 35px;
+  padding-right: 35px;
+  padding-bottom: 5px;
+}
+
+div.nextPaymentSubmission {
+  display: inline;
+}
+.nextPaymentSubmission span {
+  font-size: 12px;
+  display: inline;
+}
+
+.modal .dialog.paymentHistory {
+  padding: 0px;
+
+  .dialog-header {
+    background-color: #EEE;
+    height: 20px;
+    padding-bottom: 10px;
+  }
+
+  .dialog-body {
```

Comment:
this needs `overflow-y: scroll;` or else this happens:
![screen shot 2016-08-26 at 4 27 33 pm](https://cloud.githubusercontent.com/assets/549654/18023141/3028338a-6baa-11e6-94c2-99345d57e5e9.png)